scripts/templates_create_dagster_package/MANIFEST.in.tmpl

```diff
@@ -0,0 +1,3 @@
+include README.md
+include LICENSE
+include {{ underscore_name }}/py.typed
```

Comment:
@yuhan @smackesey  I'm not 100% sure what should go in the Manifest.in file. I based it on some other ones we have, but I also see some with `recursive-include`s. Is there a rule for what should go in here?

Reply:
Good question, I think what you have is fine. Most relevant files are included by default: https://packaging.python.org/en/latest/guides/using-manifest-in/#how-files-are-included-in-an-sdist

You only need more entries here if there's stuff like an embedded JS webapp or a directory full of data files.